falcon/api.py

```diff
@@ -736,7 +736,7 @@ def _handle_exception(self, req, resp, ex, params):
         """
 
         for err_type, err_handler in self._error_handlers:
-            if isinstance(ex, err_type):
+            if isinstance(ex, tuple(err_type)):
```

Comment:
I am probably stating the obvious, but I think we would also need to continue supporting the current case of a single exception class, not just iterables as your prototype would imply.

Reply:
sure, I will make sure of that, thanks for the reminder!